contents/security/Https.md

```diff
@@ -1,59 +1,62 @@
 # Security
 
-Authors: [Boxin]()
+Author: [Boxin](https://github.com/boxin-yang)
 
 # Overview
 
-Https is the end to end encryption on data on top of http to prevent network sniffing(eavesdropping data packets). In this tutorial, we will cover four questions to have a better understanding of https.The questions are :
+Https is the end to end encryption on data on top of http to prevent network sniffing(eavesdropping data packets). In this tutorial, we will cover four questions to have a better understanding of https. The questions are :
+
 - [Why do we need https](#why-do-we-need-https)
 - [Why https is secure](#why-https-is-secure)
 - [How to set up https](#how-to-set-up-https)
-- [Miss use of https](#misuse-of-https)
+- [Misuse of https](#misuse-of-https)
 
 ## Why do we need https
 
-The webpage application usually runs over IP network, which is vulnerable to network sniffing. The old http transmits data packet in plain text and if network is sniffed, the sniffer can see confidential information in the data packet such as password or [session token](http://searchsoftwarequality.techtarget.com/definition/session-ID). Here are some examples on how a plain text could be sniffed.
+The web application usually runs over IP network, which is vulnerable to network sniffing. The old http transmits data packet in plain text and if network is sniffed, the sniffer can see confidential information in the data packet such as password or [session token](http://searchsoftwarequality.techtarget.com/definition/session-ID). Here are some examples on how a plain text could be sniffed.
 
-- On a public free wifi, a Wifi adapter with monitor mode would be able to capture all the ongoing packets to and from the wifi access point, regardless of its destination. If the traffic is transmitted over http, the data sent over wifi is in plain text and the session token and password can be stolen. One famous example would be [Firesheep](https://github.com/codebutler/firesheep) , a Firefox plugin to sniff session token used in websites such as Facebook.This has made Facebook [change its default protocol from http to https](https://www.facebook.com/notes/facebook-engineering/secure-browsing-by-default/10151590414803920/)
+- On a public free wifi, a Wifi adapter with monitor mode would be able to capture all the ongoing packets to and from the wifi access point, regardless of its destination. If the traffic is transmitted over http, the data sent over wifi is in plain text and the session token and password can be stolen. One famous example would be [Firesheep](https://github.com/codebutler/firesheep) , a Firefox plugin to sniff session token used in websites such as Facebook. This has made Facebook [change its default protocol from http to https](https://www.facebook.com/notes/facebook-engineering/secure-browsing-by-default/10151590414803920/)
```

Comment:
* a public free wifi -> public free wifi
* a Wifi adapter with monitor mode -> a wifi adapter in monitor mode